### ISSUE TITLE
[SUPPORT] Conflict of airflow.cfg files

### ISSUE THREAD

**Issue by lecorguille** [idealista/airflow-role#104]
### Description

I'm playing the role with almost the default config.
So I have a /etc/airflow/airflow.cfg with a mysql connection parameter

But during this tasks :
https://github.com/idealista/airflow-role/blob/master/tasks/users.yml#L3-L8

An other airflow config directory is created in the root homedir with an other airflow.cfg.
Thus the following tasks use this airflow.cfg with the default sqllite.

### Additional Information

Note that I'm installing Airflow 2.1.0
And that I bypassed the installation tasks with a pre_roles
```yaml
# See https://airflow.apache.org/docs/apache-airflow/stable/installation.html#installation-tools
- name: Install some dependencies
  package:
    name: "{{ item }}"
  with_items:
    - freetds-bin
    - krb5-user
    - ldap-utils
    - libffi6
    - libsasl2-2
    - libsasl2-modules
    - libssl1.1
    - locales
    - lsb-release
    - sasl2-bin
    - sqlite3
    - unixodbc

# See https://github.com/idealista/airflow-role/blob/master/tasks/install.yml
- name: Airflow | Install pip "{{ airflow_pip_version }}" version
  pip:
    name: pip
    version: "{{ airflow_pip_version }}"
  when: airflow_pip_version is defined

- name: Airflow | Install virtualenv
  pip:
    executable: "{{ airflow_pip_executable }}"
    name: virtualenv

- name: Airflow | Check if exists virtualenv
  stat:
    path: "{{ airflow_app_home }}/pyvenv.cfg"
  register: virtualenv_check

- name: Airflow | Set a virtualenv
  become: true
  become_user: "{{ airflow_user }}"
  command: "virtualenv -p python{{ airflow_python_version | default(omit) }} {{ airflow_app_home }}"
  when: not virtualenv_check.stat.exists

- name: Airflow | Install airflow
  pip:
    name: apache-airflow
    version: "{{ airflow_version }}"
    extra_args: "--constraint '{{ contraint_url }}'"
    virtualenv: "{{ airflow_app_home }}"
```

**Comment by ultraheroe** [comment]
Hello @lecorguille

We encourage to set your `AIRFLOW_HOME` and `AIRFLOW_CONFIG` at your own discretion. (see `README.md`) this is because you may be want to change the default path, etc. So for that reason we are not setting this variables at task level.

In addition you may notice that one of the first tasks our role execute is for setting (export `AIRFLOW_HOME` and `AIRFLOW_CONFIG`) using the vars `airflow_app_home` and `airflow_conf_path`

You can see too that for testing purpose, in `molecule/converge.yml` for the converge play we are setting at play level the environment vars.

Other example in a playbook that runs our role could be:

`main.yml`
```yaml

...

- name: Install Airflow
  hosts: myAirflow
  pre_tasks:
    - include: includes/someTasks.yml
  roles:
    - role: airflow-role
      tags: airflow-role
  environment:
    AIRFLOW_HOME: "{{ airflow_app_home }}"
    AIRFLOW_CONFIG: "{{ airflow_conf_path }}/airflow.cfg"
```

I see and I suppose that in your playbook are using only some parts of our role, so It is up to you to set up the env vars at a level that you consider.

TLDR; Setting the `AIRFLOW_HOME` and `AIRFLOW_CONFIG` is up to the user and how is using our role

**Comment by ultraheroe** [comment]
closed due to inactivity

**Comment by lecorguille** [comment]
Thanks for the reply. I will investigate on that.
